torchgeo/datasets/landcoverai.py

```diff
@@ -50,7 +52,223 @@ class LandCoverAI(NonGeoDataset):
 
     If you use this dataset in your research, please cite the following paper:
 
-    * https://arxiv.org/abs/2005.02264v3
+    * https://openaccess.thecvf.com/content/CVPR2021W/EarthVision/html/Boguszewski_LandCover.ai_Dataset_for_Automatic_Mapping_of_Buildings_Woodlands_Water_and_CVPRW_2021_paper.html
```

Comment:
```suggestion
    * https://doi.org/10.1109/cvprw53098.2021.00121
```
Links to the same paper but with a shorter URL. This should fix the flake8 tests.

Reply:
I prefer open access to our paper, so I will restore the arxiv link. I don't like the IEEE paywall ;)

Reply:
Agreed. If you really want to keep the open access version, you could add `  # noqa: E501` to the closing `"""` so that flake8 ignores the long line. But arXiv is also fine with me.